Library/Subscription.cs

```diff
@@ -587,6 +519,50 @@ public RecurlyList<CouponRedemption> GetRedemptions()
             return statusCode == HttpStatusCode.NotFound ? null : coupons;
         }
 
+        /// <summary>
+        /// Request that an update to a subscription take place
+        /// </summary>
+        /// <param name="uuid">The uuid of the subscription to be changed</param>
+        /// <param name="change">A subscription change object listing what to change about the subscription</param>
+        public static Subscription ChangeSubscription(String uuid, SubscriptionChange change)
+        {
+            if (uuid.IsNullOrEmpty())
+            {
+                throw new Recurly.RecurlyException("Must have an existing subscription to make changes.");
```

Comment:
I don't know if this is the right exception here. I think something like `ArgumentException` makes more sense. We want to throw an error that the calling code would never catch. If this is null or empty the program or request should explode. The message should say something about what the problem is (that the uuid parameter is null or empty).

Reply:
Good catch. It made more sense as a `RecurlyException` when the uuid was a class property. Now, `ArgumentException` makes a lot more sense.